pom.xml

```diff
@@ -212,6 +212,16 @@
         <artifactId>feign-form</artifactId>
         <version>${feign-form.version}</version>
       </dependency>
+      <dependency>
+        <groupId>org.springframework</groupId>
```

Comment:
let's remove these spring and findbugs unwante dependencies

Reply:
removed